src/main/kotlin/com/cognifide/gradle/aem/environment/ServiceAwait.kt

```diff
@@ -0,0 +1,78 @@
+package com.cognifide.gradle.aem.environment
+
+import com.cognifide.gradle.aem.common.AemExtension
+import com.cognifide.gradle.aem.common.ProgressLogger
+import com.cognifide.gradle.aem.common.Retry
+import com.cognifide.gradle.aem.common.http.HttpClient
+import com.cognifide.gradle.aem.common.http.RequestException
+import kotlin.streams.toList
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+
+class ServiceAwait(private val aem: AemExtension) {
```

Comment:
we have `AwaitAction` ... the name is reversed :D

Reply:
Np. I have an idea for a new name.